### PR TITLE
Yama LSM

### PR DESCRIPTION
**Pull request by thypon**
#1273

### CODE REVIEW

File: srcpkgs/linux3.14/files/i386-dotconfig [hunk 1]
```diff
@@ -2019,7 +2022,7 @@ CONFIG_I2O_PROC=m
 CONFIG_MACINTOSH_DRIVERS=y
 CONFIG_MAC_EMUMOUSEBTN=m
 CONFIG_NETDEVICES=y
-CONFIG_MII=m
+CONFIG_MII=y
```
Comment (xtraeme) [inline srcpkgs/linux3.14/files/i386-dotconfig]: any reason for this?

File: srcpkgs/linux3.14/files/i386-dotconfig [hunk 2]
```diff
@@ -3085,7 +3088,7 @@ CONFIG_HSI_CHAR=m
 #
 # PPS support
 #
-CONFIG_PPS=m
+CONFIG_PPS=y
```
Comment (xtraeme) [inline srcpkgs/linux3.14/files/i386-dotconfig]: and this one?

Comment (thypon) [inline srcpkgs/linux3.14/files/i386-dotconfig]: It was enabled when i put yama by default. Some dependency i think.

File: srcpkgs/linux3.14/files/i386-dotconfig (hunk 1 above)
Comment (thypon) [inline srcpkgs/linux3.14/files/i386-dotconfig]: Autoadded by menuconfig when i enabled Yama LSM, not intended, but a side effect. It happens only on 3.14, we could exclude it.

**Comment by xtraeme** [comment]
Ok, but this PR cannot be applied. Try to rebase it :-)

**Comment by xtraeme** [comment]
And while you are at it, please update linux3.18 as well.

**Comment by thypon** [comment]
timeout

**Pull request merged**